component/doc/README.md

```diff
@@ -0,0 +1,346 @@
+ 
+ 2. Create a new application as described in below.
+ 
+    a. Navigate to the following URL: `https://www.linkedin.com/developer/apps/new`
+    
+    b. Enter the required details.
+    
+        * App Name and Company details. The company should have a linkedin page.
+        * Upload an image that you wish to use at the company logo.
+        * Select the checkbox to agree to the LinkedIn terms and conditions.
+    c. Click on **Create App**. You will redirect to a page with **Settings** for the created application. 
+    
+    ![linkedin-app-create-page](img/linkedin-app-create-page.png)
+    
+    ![linkedin-app-settings-page](img/linkedin-app-settings-page.png)
+    
+    d. Navigate to the **Auth** tab which you can see the **Client ID** and
+     **Client Secret**.
+     
+    e. Enter the **Redirect URL** under the **OAuth 2.0 settings** as the following format and click on **Update
+    **. `https://{hostname}:{port}/commonauth`. The default redirect URL in WSO2 Identity Server is `https
+    ://localhost:9443/commonauth`.
+    
+    ![linkedin-app-auth-page](img/linkedin-app-auth-page.png)
+    
+    f. Copy the **Client ID** and **Client Secret** from the resulting page.
+    
+##Step 2 - Deploy the Pickup Dispatch Sample Web App
+
+The next step is to deploy the Pickup Dispatch sample web app in order to use it in this scenario.
+
+To configure this, see [saml2-web-app-pickup-dispatch 
+webapp](https://is.docs.wso2.com/en/5.9.0/learn/deploying-the-sample-app/#deploying-the-saml2-web-app-pickup-dispatch-webapp).
+
+> :information_source: Please avoid **Configuring the service provider** section in the above reference because we will
+> configure it in this document with some other configurations.
+
+##Step 3 - Configure the Identity Provider (IdP)
+
+Now you have to configure WSO2 Identity Server, which was started in step 01 by [adding a new identity 
+provider](https://is.docs.wso2.com/en/5.9.0/learn/adding-and-configuring-an-identity-provider/#adding-and-configuring-an-identity-provider).
+
+1. Log in to the [Management Console](https://is.docs.wso2.com/en/5.9.0/setup/getting-started-with-the-management-console/) 
+as an administrator.
+
+2. In the **Identity Providers** section under the **Main** tab of the management console, click **Add**.
+
+3. Enter a suitable name as the **Identity Provider Name** (e.g., LinkedIn).
+    >As our resident Identity Provider is
+ WSO2 IS, the Alias will appear as follows - `https://(host-name):(port)/oauth2/token`
+
+4. Optionally, you can add the LinkedIn public certificate by uploading it. You can do this by selecting the **Upload
+ IDP certificate** option from **Choose IDP certificate type** and click on the **Choose file** button next to the
+  **Identity Provider Public Certificate** field. Then upload the certificate from a local directory. 
+  
+    Some browsers let us download the public certificate. If not you can skip this step.
```

Comment:
Some browsers alloy you to download the public certificate. If not, you can skip this step.